src/pages/storage/forms/StorageVolumeFormMenu.tsx

```diff
@@ -76,14 +77,14 @@ const StorageVolumeFormMenu: FC<Props> = ({
                 disableReason={disableReason}
               />
               {contentType === "filesystem" &&
-                ["zfs", "lvm", "ceph"].includes(poolDriver) && (
+                [zfsDriver, lvmDriver, cephDriver].includes(poolDriver) && (
```

Comment:
Good idea to use the constants! Probably good to have this array in the `storageOptions.tsx` as well, some name like `export const driversWithFilesystemSupport = [...]`?

Reply:
event better! :+1: